cloudbuild.yaml

```diff
@@ -14,10 +14,118 @@
 
```

Comment:
I think it might be use to, in a sentence or two, describe the purpose of this document.

Can this be triggered manually? If yes, then maybe provide an example gcloud command? For example: https://cloud.google.com/cloud-build/docs/running-builds/start-build-manually#running_builds

Something like...

Command commands:
    Push to staging: gcloud ...
    Push to prod: gcloud ...

Reply:
The file can be triggered manually with `gcloud builds submit --config cloudbuild.yaml .` and I'll add some comments at the doc explaining the purpose of the file. Also just to make sure I'm on the right page, my idea of staging vs. prod right now is that the dev branch corresponds 1-1 with staging, and the prod branch corresponds to prod. I think right now the idea of environments are also highly coupled with GitHub branches, so to push to different environments manually, you would have to make sure that you're on the appropriate branch. I know this is kind of confusing so I'm happy to maybe discuss this in more detail over gvc if you're free.

Reply:
Thanks for adding the documentation, I think that it's very useful. A couple small suggestions:

1. In the first paragraph, maybe add a link to the cloud build documentation, for people who have never heard of cloud build before?
2. For the explanation on dev, prod, versus some other branch, it might be clearer to explain the difference along the lines of:

> If Cloud Build is ran from a git branch named either "dev" or "prod", the configuration will be applied to the cloud project provided via environment variable $PROJECT_ID. Otherwise, the changes will be planned, but not applied (dry run mode, which is useful when testing config changes from a development branch).

Or something along those lines.